test/test_losses.py

```diff
@@ -9,6 +9,71 @@
 from torch.testing import assert_allclose
 
 
+class TestFocalLossWithLogits:
+    def test_smoke_none(self, device):
+        num_classes = 1
+        logits = torch.rand(2, num_classes, 3, 2).to('cpu')
```

Comment:
why you send to cpu ? do this: `logits = torch.rand(2, num_classes, 3, 2, device=device, dtype=dtype)`